docs/reference/configuration.md

```diff
@@ -107,8 +107,8 @@ This reference uses `.` to denote the nesting of values.
 + `email.username` `(string)` - A string representing the username of the specific account at the SMTP. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.password` `(string)` - A string representing the password of the account. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.sender` `(string)` - A string representing the email account which is set as the sender.
-+ `email.rate_limit` `(float)` - A float representing the amout emails that can be sent per second.
-+ `email.burst` `(int)` - An integer representing the maximum amount of credits that can be obtained.
++ `email.limit_every` `(Duration)` - Average time in seconds to wait between sending emails.
```

Comment:
But the actual type is `float64`!  As suggested, please use `time.Duration` and code the conversion.  Also our config already has a way to implement default values; please use it.